examples/sshd.yml

```diff
@@ -22,7 +22,8 @@ services:
     image: "linuxkit/sshd:abc1f5e096982ebc3fb61c506aed3ac9c2ae4d55"
 files:
   - path: root/.ssh/authorized_keys
-    contents: '#your ssh key here'
+    source: ~/.ssh/id_rsa.pub
```

Comment:
I think CI is failing to build this example because it can't find this on its host?